Dangerfile

```diff
@@ -0,0 +1,4 @@
+# Don't let testing shortcuts get into master by accident
+
+fail("fdescribe left in tests") if `grep -r fdescribe spec/`.length > 1
```

Comment:
got options on better ways to pull off this grep? ( not being cynical, would be interested on ways to do it better for Danger in it's entirety )